defaults/main.yml

```diff
@@ -13,5 +13,13 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
-letsencrypt_certbot_args: []
+# certbot arguments
+letsencrypt_certbot_args: ["certonly",
```

Comment:
The default is exactly as you had it @jaywink -- so if people don't care, they don't have to change it. This was an itch I had...